src/hotspot/cpu/aarch64/c1_LIRAssembler_aarch64.cpp

```diff
@@ -1194,13 +1194,17 @@ void LIR_Assembler::emit_alloc_obj(LIR_OpAllocObj* op) {
     add_debug_info_for_null_check_here(op->stub()->info());
     __ br(Assembler::NE, *op->stub()->entry());
   }
-  __ allocate_object(op->obj()->as_register(),
-                     op->tmp1()->as_register(),
-                     op->tmp2()->as_register(),
-                     op->header_size(),
-                     op->object_size(),
-                     op->klass()->as_register(),
-                     *op->stub()->entry());
+  address tpc = __ allocate_object(op->obj()->as_register(),
+                                   op->tmp1()->as_register(),
+                                   op->tmp2()->as_register(),
+                                   op->header_size(),
+                                   op->object_size(),
+                                   op->klass()->as_register(),
+                                   *op->stub()->entry());
+  if (tpc == nullptr) {
+    bailout("trampoline stub overflow");
```

Comment:
The message is confusing. How about `"no space for trampoline stub"` used in `c1_LIRAssembler_ppc.cpp`.

Reply:
Makes sense. Fixed.